ifrs17-template/Test/IfrsVariablesTest.ipynb

```diff
@@ -148,6 +196,12 @@
                 "\n                              AccidentYear = Int32.TryParse((row.Field<string>(nameof(IfrsVariable.AccidentYear))), out var accidentYear) ? accidentYear : (int?)null,",
```

Comment:
instead of all this code, can't you use you the TryParse?

Reply:
@dcolleoni I don't understand your suggestion. Please explain better. AccidentYear must be an int? type, so it can be null thus there is this Int32.TryParse.

Reply:
don't change the AccidentYear. I just wanted to point out this line that could serve you for inspiration for the your new code about ValueN

Reply:
Ok, thank you @dcolleoni